frontend/src/component/admin/instance-privacy/InstancePrivacySection.tsx

```diff
@@ -0,0 +1,211 @@
+import { Box, styled } from '@mui/material';
+import { Badge } from 'component/common/Badge/Badge';
+import ClearIcon from '@mui/icons-material/Clear';
+import CheckIcon from '@mui/icons-material/Check';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { TooltipLink } from 'component/common/TooltipLink/TooltipLink';
+
+const StyledContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    padding: theme.spacing(3),
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: theme.shape.borderRadiusLarge,
+}));
+
+const StyledCardTitleRow = styled(Box)(() => ({
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+}));
+
+const StyledCardDescription = styled(Box)(({ theme }) => ({
+    color: theme.palette.text.secondary,
+    fontSize: theme.fontSizes.smallBody,
+    marginTop: theme.spacing(2),
+}));
+
+const StyledPropertyName = styled('p')(({ theme }) => ({
+    display: 'table-cell',
+    fontWeight: theme.fontWeight.bold,
+    paddingTop: theme.spacing(2),
+}));
+
+const StyledPropertyDetails = styled('p')(({ theme }) => ({
+    display: 'table-cell',
+    paddingTop: theme.spacing(2),
+    paddingLeft: theme.spacing(4),
+}));
+
+const StyledDataCollectionExplanation = styled('div')(({ theme }) => ({
+    display: 'table-cell',
+    width: '75%',
+    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(2),
+}));
+
+const StyledDataCollectionBadge = styled('div')(({ theme }) => ({
+    display: 'table-cell',
+}));
+
+const StyledTag = styled('span')(({ theme }) => ({
+    display: 'block',
+    textAlign: 'right',
+    color: theme.palette.neutral.dark,
+}));
+
+const StyledDescription = styled('div')(({ theme }) => ({
+    maxWidth: theme.spacing(50),
+    color: theme.palette.text.secondary,
+    fontSize: theme.fontSizes.smallBody,
+}));
+
+const StyledDataCollectionPropertyRow = styled('div')(() => ({
+    display: 'table-row',
+}));
+
+const StyledDataCollectionPropertyTable = styled('div')(() => ({
+    display: 'table',
+}));
+
+const StyledDataCollectionPropertyCell = styled('div')(() => ({
+    display: 'table-cell',
+}));
+
+interface IPrivacyProps {
+    title: string;
+    infoText: string;
+    concreteDetails: Record<string, string>;
+    enabled: boolean;
+    changeInfoText: string;
+    variablesText: string;
+    dependsOnText?: string;
+}
```

Comment:
Maybe move this interface above the `InstancePrivacySection` component and rename it `IInstancePrivacySectionProps`